pedrovgs/KotlinSnapshot
# 70 fail on missing snapshot

### :pushpin: References
* **Issue:** #70 

### :tophat: What is the goal?

Allow to run KotlinSnapshot in a mode where it fails on missing snapshots.

### :memo: How is it being implemented?

- I have introduced a new property/env variable `failOnMissingSnapshots`. If this is set to `1`, the camera will fail instead of writing the snapshot.

### :boom: How can it be tested?

- Write a snapshot test with missing snapshot, run `failOnMissingSnapshots=1 ./gradlew test` – The test should fail.
- Run `./gradlew test` – The snapshot should be created and the test should pass
- Run `failOnMissingSnapshots=1 ./gradlew test` – The test still should pass.

See also: `CameraTest`.
